### DISCUSSION TITLE
Flatpak

### DISCUSSION THREAD

**Discussion by quantum-quirks** [andyholmes/valent#717] [Distribution Packaging]
Are there any plans to have a flatpak stable release version? If so when?
We can't have only flatpak nightly indefinitely.

**Comment by andyholmes** [comment]
This is a personal project I work on in my spare time, so there are no ETAs or timelines for when it may be considered stable and released.

**Comment by henry88g** [comment]
FYI

Trying to install from Flatpak on Ubuntu 22.04, I got the following:
Unable to Install Valent
White pulling app/ca.andyholmes.Valent/x86_64/master from remote valent-origin: Server returned status 503: Service Unavailable 
The gnome icon appeared in my top toolbar.
Installation went to 99%

**Reply by andyholmes** [reply]
I believe the nightly Flatpak repository is being throttled by GitHub, so the server times out sometimes or gets very slow. You'll just have to retry or download a Flatpak bundle if the service is unavailable.
